packages/ckeditor5-table/tests/converters/downcast.js

```diff
@@ -606,132 +549,104 @@ describe( 'downcast converters', () => {
 	} );
 
 	describe( 'downcastInsertCell()', () => {
-		// The beforeEach is duplicated due to ckeditor/ckeditor5#6574. New test are written using TableEditing.
-		beforeEach( () => {
-			return VirtualTestEditor.create()
-				.then( newEditor => {
-					editor = newEditor;
-					model = editor.model;
-					doc = model.document;
-					root = doc.getRoot( 'main' );
-					view = editor.editing.view;
-
-					defaultSchema( model.schema );
-					defaultConversion( editor.conversion );
-				} );
-		} );
+		// The insert table cell downcast conversion is not executed in data pipeline.
```

Comment:
Maybe we should add test to make sure that cell conversion is not used in data pipeline?

Reply:
As above.